index.html

```diff
@@ -2,16 +2,17 @@
 <html lang="en">
 <head>
 	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width,initial-scale=1, user-scalable=no">
```

Comment:
Are you sure we should not scale the page on the mobile devices? (`user-scalable=no`)